src/AzureIoTHub.Portal.Shared/Models/v1.0/IdeaRequest.cs

```diff
@@ -12,5 +12,7 @@ public class IdeaRequest
 
         [Required]
         public string Body { get; set; } = null!;
+
+        public bool Consent { get; set; }
```

Comment:
Should be better to rename this property

```suggestion
        public bool ConsentToCollectTechnicalDetails { get; set; }
```